app/src/main/java/io/appium/uiautomator2/server/ServerInstrumentation.java

```diff
@@ -78,24 +78,30 @@ private void setAccessibilityServiceState() {
 
     private ServerInstrumentation(Context context, int serverPort, int mjpegServerPort) {
         if (!isValidPort(serverPort)) {
-            throw new UiAutomator2Exception(String.format(
-                "The server port is out of valid range [%s;%s]: %s",
+            Logger.warn(String.format(
```

Comment:
warn is deprecated. Please use `warning`

Reply:
`warning` doesn't seem to be defined.

Reply:
ah, nvm, just ignore them

Reply:
Logger is generally referring to: app/src/main/java/io/appium/uiautomator2/utils/Logger.java which currently only has a `warn`.

Reply:
yep, I have forgotten that this project uses a custom logging wrapper